src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor

```diff
@@ -278,4 +278,4 @@
         IsOpenInPosition = true;
         position = positionValue;
     }
-}
```

Comment:
revert this change plz, do not remove this empty new line at the end of the file.

Reply:
Hello, ok.

Reply:
you didn't change anything!